javascript/javascript2/week3/presser.js

```diff
@@ -0,0 +1,99 @@
+
+/* var confettiSettings = { target: 'my-canvas' };
+var confetti = new ConfettiGenerator(confettiSettings);
+confetti.render(); */
+
```

Comment:
if you want to add confetti to your code first you can copy the script link from the homework link and paste it into before calling the main js file.and also you can add the javascript code into after the below line.
 document.getElementById('winner').innerHTML='L is winner <br> Score<br>'+score;and also  
document.getElementById('winner').innerHTML='S is winner <br> Score<br>'+score; and here also the same  with different canvas id.